frontend/src/router/router.js

```diff
@@ -748,11 +748,6 @@ const routerOptions = [
       title: "Login"
     })
   },
-  {
-    path: '/user/account/password/set',
```

Comment:
We would still want this (and the component) for an existing user to change their password right?

Reply:
Hey Anthony! Yeah, I was fixing tests and noted